pkg/crawl/crawler.go

```diff
@@ -76,6 +84,8 @@ func (c Crawler) Crawl(ctx context.Context, root string, match MatchFunc, proces
 		default:
 		}
 		if d.IsDir() {
+			fmt.Println(path)
+			limiter.Take()
```

Comment:
Should this go inside the if block in the line below so we don't limit ourself on directories that are ignored anyway?